### PR TITLE
ES-829615-Add-Subpage-for-Azure-App-Service-WordtoPDF

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Added Subpages of Azure Function for Conver Word to PDF.

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    }
+    return View("Index");
+}
+
+/// <summary>
+/// Sets the alternate font when a specified font is not installed in the production environment
+/// </summary>
+/// <param name="sender"></param>
+/// <param name="args"></param>
+private void FontSettings_SubstituteFont(object sender, SubstituteFontEventArgs args)
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md:195]: Remove font handling related code.

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+![Click the Publish button](Azure_Images/App_Service_Linux/Before_Publish_WordtoPDF.png)
+
+Step 10: Now, Publish has been succeeded.
+![Publish has been succeeded](Azure_Images/App_Service_Linux/After_Publish_WordtoPDF.png)
+
+Step 11: Now, the published webpage will open in the **browser**. Select the Word document and Click **Convert to PDF** to convert the given Word document to a PDF.You will get the output PDF document as follows.
+
+![Output image PDF document](WordToPDF_images/WordToPDF_Output_Cloud.png)
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v1).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md:263]: Change the link

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats-toc.html [hunk 3]
```diff
@@ -250,6 +250,12 @@
 							<li>
 								<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-MAC">Mac</a>
 							</li>
+							<li>
+								<a href="/file-formats/DocIO/Azure-MainPage">Azure</a>
+								<ul>
+									<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux">App Service(Linux)</a>
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats-toc.html:256]: Updated

File: File-Formats/DocIO/Azure-MainPage.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Nuget package required
+<table>
+<thead>
+<tr>
+<th>
+Azure Services<br/></th><th>
+NuGet package name<br/></th></tr></thead>
+<tr>
+<td>
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Azure-MainPage.md:27]: Updated

File: File-Formats/DocIO/Azure-MainPage.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+App Service (Windows)<br/></td><td>
+{{'[Syncfusion.DocToPDFConverter.AspNet](https://www.nuget.org/packages/Syncfusion.DocToPDFConverter.AspNet)' |markdownify}}<br/></td></tr>
+<tr>
+<td>
+App Service (Linux)<br/></td><td>
+{{'[Syncfusion.DocIORenderer.Net.Core](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core)' | markdownify}}<br/>
+{{'[SkiaSharp.NativeAssets.Linux](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux)' | markdownify}}<br/>{{'[HarfBuzzSharp.NativeAssets.Linux](https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.Linux)' |markdownify}} <br/></td></tr>
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Azure-MainPage.md:34]: Updated

File: File-Formats/DocIO/Azure-MainPage.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+<tr>
+<td>
+Azure Functions v1 (.NET Framework)<br/></td><td>
+{{'[Syncfusion.DocToPDFConverter.AspNet](https://www.nuget.org/packages/Syncfusion.DocToPDFConverter.AspNet)' |markdownify}} <br/></td></tr>
+<tr>
+<td>
+Azure Functions v4 (.NET 6.0, 7.0)<br/></td><td>
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Azure-MainPage.md:41]: updated

File: File-Formats/DocIO/Azure-MainPage.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+App Service (Windows)<br/></td><td>
+{{'[Syncfusion.DocToPDFConverter.AspNet](https://www.nuget.org/packages/Syncfusion.DocToPDFConverter.AspNet)' |markdownify}}<br/></td></tr>
+<tr>
+<td>
+App Service (Linux)<br/></td><td>
+{{'[Syncfusion.DocIORenderer.Net.Core](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core)' | markdownify}}<br/>
+{{'[SkiaSharp.NativeAssets.Linux](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux)' | markdownify}}<br/>{{'[HarfBuzzSharp.NativeAssets.Linux](https://www.nuget.org/packages/HarfBuzzSharp.NativeAssets.Linux)' |markdownify}} <br/></td></tr>
+<tr>
+<td>
+Azure Functions v1 (.NET Framework)<br/></td><td>
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Azure-MainPage.md:37]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+Step 1: Create a new ASP.NET Core Web App(Model-View-Controller).
+![Create a ASP.NET Core Web App project](Azure_Images/App_Service_Linux/Create-Project-WordtoPDF.png)
+
+Step 2: Create a project name and select the location.
+![Configure your new project](Azure_Images/App_Service_Linux/Configure_Project_WordtoPDF.png)
+
+Step 3: Click **Create** button
+![Additional Information](Azure_Images/App_Service_Linux/Additional_Information_WordtoPDF.png)
+
+Step 4: Install the following **Nuget packages**.
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md:24]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md [hunk 9]
```diff
@@ -0,0 +1,263 @@
...
+
+* [Syncfusion.DocIORenderer.Net.Core](https://www.nuget.org/packages/Syncfusion.DocIORenderer.Net.Core) 
+* [SkiaSharp.NativeAssets.Linux](https://www.nuget.org/packages/SkiaSharp.NativeAssets.Linux)
+
+NuGet package as a reference to your .NET Core application from [Nuget.org](https://www.nuget.org/).
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md:29]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md:195]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md [hunk 10]
```diff
@@ -0,0 +1,263 @@
...
+
+Syncfusion  DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in Azure App service on Linux**
+
+## Steps to convert Word document to PDF in Azure App Service on Linux:
+
+Step 1: Create a new ASP.NET Core Web App(Model-View-Controller).
+![Create a ASP.NET Core Web App project](Azure_Images/App_Service_Linux/Create-Project-WordtoPDF.png)
+
+Step 2: Create a project name and select the location.
+![Configure your new project](Azure_Images/App_Service_Linux/Configure_Project_WordtoPDF.png)
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-Azure-App-Service-Linux.md:19]: Updated

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**